Trigger/python/genTriggerFcl.py

```diff
@@ -25,13 +25,13 @@
 
 def appendEpilog(trig_path, relProjectDir, outDir, srcDir, verbose, doWrite, sourceFiles, targetFiles):
 
-    trk_filters      = ['EventPrescale','SDCountFilter','TCFilter', 'HSFilter', 'TSFilter']
-    helix_filters    = ['EventPrescale','SDCountFilter','TCFilter', 'HSFilter']
-    tc_filters       = ['EventPrescale','SDCountFilter','TCFilter']
```

Comment:
I have seen a big cosmic ray shower light up an entire detector with many thousands of hits.  Are we sure we want to remove this filter?  How much does it cost to keep it?

Reply:
keeping them would not fit the model of the Intensity stream I've been developing as it will prevent reco modules run in the early stage to evaluate those quantities. I think that we need to add protections to those modules. The events you are referring to I don't expect to be that frequent... worst case scenario the DEBUG stream implementation will keep us safe.